prac 5/state_names.py

```diff
@@ -4,7 +4,7 @@
 File needs reformatting
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+
 CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
```

Comment:
Instead of using your if statements for finding the max length you can use the max function.
"longest_word = max((len(word) for word in list_words))"

Otherwise nice formatting. Looks good